hrabsence/CRM/HRAbsence/BAO/HRAbsenceType.php

```diff
@@ -117,6 +77,67 @@ public static function create($params) {
   }
 
   /**
+   * Creates Activity Types based on $params and $activityTypes
+   *
+   * @param  array $activityTypes
+   * @param  array   $params
+   * @param  boolean $isDebit  ('true' for 'debit' and 'false' for 'credit')
+   *
+   * @return array Created Activity Type as array
+   */
+  public static function createActivityType($activityTypes, $params, $isDebit) {
+    $weight = count($activityTypes["values"]);
+    if ($isDebit) {
+      $valueId = $params['debit_activity_type_id'];
+      $valueLabel = $params['title'];
+    } else {
+      $valueId = $params['credit_activity_type_id'];
+      $valueLabel = ts('%1 (Credit)', array(1 => $params["title"]));
+    }
+
+    $paramsCreate = array(
+      'weight' => ($weight+1),
+      'label' => $valueLabel,
+      'filter' => 1,
+      'is_active' => 1,
+      'is_optgroup' => 0,
+      'is_default' => 0,
+      'grouping' => 'Timesheet'
+    );
+
+    return civicrm_api3('activity_type', 'create', $paramsCreate);
+  }
+
+  /**
+   * Updates Option Value based on $params
+   *
+   * @param  array $params
+   * @param  boolean $isDebit  ('true' for 'debit' and 'false' for 'credit')
+   *
+   * @return void
```

Comment:
Sorry, I missed this on my last review. If the method doesn't return anything, you should ommit the @return